### ISSUE TITLE
Path not removed & ssl redirect for default backend

### ISSUE THREAD

**Issue by cubed-it** [Azure/application-gateway-kubernetes-ingress#490]
I have set up aks etc according to the tutorial. I also have rbac active and shared to true. Version 0.8.0.

[1] I did the whole thing twice and on the second run I created the following problem: On the second run I directly specified a path, as it is called "Integrate with other services" - see below. 
Now the ssl redirect only works for the path but not for the default backend.

[2] If I remove the path again and specify only one backend, the path will not be removed. If I change the path, the change is applied.

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: guestbook
  annotations:
    kubernetes.io/ingress.class: azure/application-gateway    
    appgw.ingress.kubernetes.io/ssl-redirect: "true"
    appgw.ingress.kubernetes.io/backend-path-prefix: "/"
spec:
  tls:
    - secretName: tlscer
  rules:
  - http:
      paths:
      - path: /hello/*
        backend:
          serviceName: frontendalt
          servicePort: 80
      - backend:
          serviceName: frontend
          servicePort: 80
```

**Comment by akshaysngupta** [comment]
@cubed-it 
1. I was able to reproduce this. This is bug that we fixed in https://github.com/Azure/application-gateway-kubernetes-ingress/pull/422. We aim to release this soon
1. I wasn't able to reproduce this.

**Comment by akshaysngupta** [comment]
@cubed-it We just released 0.9.0-rc1.
https://github.com/Azure/application-gateway-kubernetes-ingress/releases/tag/0.9.0-rc1
Please give this a shot.

**Comment by delqn** [comment]
@cubed-it Have you had a chance to try the latest version?  Today we released [0.9.0-rc2](https://github.com/Azure/application-gateway-kubernetes-ingress/releases/tag/0.9.0-rc2) - please let us know if an upgrade solved the issue.

I'm closing the issue, but would love to hear from you!
